### DISCUSSION TITLE
SESI2 - Search Input API | Next JS 2023

### DISCUSSION THREAD

**Discussion by fahmiimaniar12** [deaafrizal/forum#60] [Troubleshooting Room]
![image](https://github.com/deaaprizal/forum/assets/75206369/69c2df4f-f440-45b1-b810-e90cb9f00f60)

mau bertanya. kenapa button searchnya biasa bisa, biasa gk bisa ya ?

**Comment by danielpsrb** [comment]
halo fahmi, pastikan kode di handleSearch nya tidak ada yang salah dan pemanggilan API nya, coba di cek lgi ya

**Comment by RASZZZKING** [comment]
aku awalnya tinggal dibungkus tag form doang, eh pas udah sampe sesi3 taunya beda hehe gapapa juga sih yg penting kan fungsi dan udah berusaha

**Comment by Ham279** [comment]
<img width="590" alt="image" src="https://github.com/deaaprizal/forum/assets/174991346/18fc9e66-9315-401a-b883-ecab8c02e0e2">
<p>bang setelah saya kompail muncul pesan seperti yang dibawah</p>
<img width="600" alt="image" src="https://github.com/deaaprizal/forum/assets/174991346/92f7bd79-3ba7-4c64-9e11-4bcbcaa862ff">
<p>ini kenapa yah bang?</p>
